### PR TITLE
change line breaks, move istio document

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- move istio doc to service mesh
- change line breaks in Observability overview
- ...

**Related issue(s)**

### CODE REVIEW

File: docs/overview/51-telemetry-observability-overview.md [hunk 1]
```diff
@@ -3,15 +3,13 @@ title: Telemetry and Observability
...
-Of course, you'll want to view and analyse the data you're collecting. This is where **observability** tools come in. 
+Out of the box, Kyma provides tools to collect and expose **telemetry** data, such as metrics, traces, and log data. Of course, you'll want to view and analyse the data you're collecting. This is where **observability** tools come in. 
 
 ## Collecting data
 
-The [OpenTelemetry](https://opentelemetry.io/) observability framework is the core tool where all of Kyma's raw data comes together. 
+The [OpenTelemetry](https://opentelemetry.io/) observability framework is the core tool where all of Kyma's raw data comes together.  Among the data sources that flow into the OpenTelemetry collector are Kubernetes and Istio.
 
-Among the data sources that flow into the OpenTelemetry collector are Kubernetes and Istio.
+> **Note** All metrics relevant for observing the in-cluster Istio Service Mesh are collected separately. You can find more information about it on [Istio monitoring](link-to-istio-monitoring).
```
Comment (majakurcius) [inline docs/overview/51-telemetry-observability-overview.md:12]:
```suggestion
> **NOTE:** All metrics relevant for observing the in-cluster Istio Service Mesh are collected separately. You can find more information about it on [Istio monitoring](link-to-istio-monitoring).
```

Comment (kazydek) [inline docs/overview/51-telemetry-observability-overview.md:12]: You can find more information about it on [Istio monitoring](link-to-istio-monitoring). -> **in** Istio monitoring **(documentation**) ?

File: docs/overview/51-telemetry-observability-overview.md [hunk 2]
```diff
@@ -3,15 +3,13 @@ title: Telemetry and Observability
...
-Of course, you'll want to view and analyse the data you're collecting. This is where **observability** tools come in. 
+Out of the box, Kyma provides tools to collect and expose **telemetry** data, such as metrics, traces, and log data. Of course, you'll want to view and analyse the data you're collecting. This is where **observability** tools come in. 
 
 ## Collecting data
 
-The [OpenTelemetry](https://opentelemetry.io/) observability framework is the core tool where all of Kyma's raw data comes together. 
+The [OpenTelemetry](https://opentelemetry.io/) observability framework is the core tool where all of Kyma's raw data comes together.  Among the data sources that flow into the OpenTelemetry collector are Kubernetes and Istio.
 
-Among the data sources that flow into the OpenTelemetry collector are Kubernetes and Istio.
+> **NOTE:** All metrics relevant for observing the in-cluster Istio Service Mesh are collected separately. You can find more information about it on [Istio monitoring](link-to-istio-monitoring).
```
Comment (NHingerl) [inline docs/overview/51-telemetry-observability-overview.md:12]:
```suggestion
> **NOTE:** All metrics relevant for observing the in-cluster Istio Service Mesh are collected separately. You can find more information about it in the [Istio monitoring documentation](link-to-istio-monitoring).
```

File: docs/overview/51-telemetry-observability-overview.md (hunk 1 above)
Comment (NHingerl) [reply docs/overview/51-telemetry-observability-overview.md]: good point, thanks!

**Review by kazydek — APPROVED** [review]

**Pull request merged**